TeaDiary/App_Start/AutofacConfig.cs

```diff
@@ -9,6 +13,12 @@ public static void AddApplicationServices(this ContainerBuilder builder)
             //Register CONTROLLER as default INTERFACE realization
```

Comment:
I think you mean INTERFACE and IMPLEMENTATION. Also, the correct word for "realization" is "implementation"

Reply:
Was guessing how to call it properly. Thanks!